addons/website_sale/views/templates.xml

```diff
@@ -1901,4 +1902,9 @@
         </xpath>
     </template>
 
+    <template id="add_to_cart_redirect" inherit_id="website.layout" name="Cart Redirection">
+        <xpath expr="//html" position="before">
+            <t t-set="html_data" t-value="dict(html_data, **{'data-add2cart-redirect': website.cart_add_on_page or '0'})"/>
```

Comment:
==
`    <t t-set="_" t-value="html_data.update({'data-add2cart-redirect': int(website.cart_add_on_page)})"/>`

And it is strange to use True vs '0', why not us int(bool) instead, or just False ?